.ci/tweet-releasenotes.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+echo "PULL_REQUEST:""$PULL_REQUEST"
+if [[ $PULL_REQUEST =~ ^([0-9]+)$ ]]; then
+  echo "Build is not for Pull Request";
+  sleep 5;
+  exit 0;
+fi
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+cd .ci-temp/checkstyle
+LATEST_RELEASE_TAG=$(git describe --tags --abbrev=0)
```

Comment:
Example of commit that is created by mvn release:prepare - https://github.com/checkstyle/checkstyle/commit/a6ee9b43d012c1d6476148974cdb1be7fc27c313
you might notice it has tag on it.
so it will be same as `git describe --tags --abbrev=0` am I right ?

Reply:
This command looks like return latest created tag.  But https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams.md you can see that we will create tag right before new release.
Please test it on local.
I still think it is better to get latest release from GitHub by API call.

Reply:
Is this what you are expecting?
```
$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
| grep -oP '"tag_name": "\K(.*)(?=")')
```

![szvsdvds](https://user-images.githubusercontent.com/71710042/174773794-b28bc7a2-70e9-4de8-878a-30e2eb598f45.png)

Reply:
Yes, but please use jq tool for parsing

Reply:
Changes Done